uber/server.py

```diff
@@ -11,6 +11,25 @@ def _add_email():
 cherrypy.tools.add_email_to_error_page = cherrypy.Tool('after_error_response', _add_email)
 
 
+def on_cherrypy_before_error_response(debug=False):
+    """
+    Write the request headers, and the last error's traceback to the cherrypy error log.
+    Do this all one line so all the information can be collected by external log collectors and easily displayed.
```

Comment:
I thought about that yea, and we can add to this list easily if we ever want that info.  It is probably a good idea to do, and maybe we can filter out the password fields.  This also displays the session_id which would allow someone to spoof a session, and also can display attendee_id's which can be used to transfer badges.   We might want to move the channel in slack that is displaying this info into a private channel